libqtile/confreader.py

```diff
@@ -122,7 +122,7 @@ def load(self):
         sys.path.insert(0, path.parent.as_posix())
 
         if name in sys.modules:
-            self._reload_config_submodules(path)
+            # self._reload_config_submodules(path)
```

Comment:
This borks my config :( I need to always restart, just like those good old times :)

Otherwise, this works perfectly. I tried adding the line and I saw some weird flickers and remains of what had already been drawn on the last reload. Is that why you commented it out?

Reply:
Argh. That's not meant to be in there! Sorry.